core/create.go

```diff
@@ -47,32 +50,18 @@ func CreateProject(path string, options CreateProjectOptions) error {
 		return ErrProjectExists
 	}
 
-	if path != "." {
-		if err := os.RemoveAll(path); err != nil {
-			return err
+	err := filepath.Walk(path, func(path string, info os.FileInfo, err error) error {
+		if os.IsExist(err) {
+			return nil
 		}
-	} else {
-		err := filepath.Walk(path, func(path string, info os.FileInfo, err error) error {
-			// RemoveAll removes nested directory in first iteration which causes
-			// os.PathError saying "no such file or directory" for next recursion of
-			// WalkFunc.
-			if os.IsNotExist(err) {
-				return nil
-			}
-			if path != "." {
-				if info.IsDir() {
-					// Remove nested non-empty directories as os.Remove() only removes
-					// files and empty directories
-					return os.RemoveAll(path)
-				} else {
-					return os.Remove(path)
-				}
-			}
+		if strings.Contains(path, gitDirectory) {
```

Comment:
That doesn't work as before it walks the git folder it already walked the root-path which already called a RemoveAll and therefore already removed also the .git folder.